loader/src/main/java/net/neoforged/fml/ISystemReportExtender.java

```diff
@@ -5,14 +5,39 @@
 
 package net.neoforged.fml;
 
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.Nullable;
+
 import java.util.function.Supplier;
 
-public interface ISystemReportExtender extends Supplier<String>
-{
-    String getLabel();
+@ApiStatus.OverrideOnly
+public interface ISystemReportExtender extends Supplier<String> {
+    /**
+     * {@return the label of the crash report value to add}
```

Comment:
This isn't for the crash report but the _system_ report, as evidenced by the name of the class. 

While crash reports have a system report, a system report can be generated independently of a crash report -- see `PerfCommand` (`/perf`) where it saves a system report alongside the performance metrics report.